.circleci/config.yml

```diff
@@ -17,7 +17,7 @@ jobs:
       - checkout
       - run:
           name: Generate combined build.gradle file for cache key
-          command: cat build.gradle */build.gradle .circleci/gradle.properties > deps.txt
+          command: cat build.gradle */build.gradle .circleci/gradle.properties .circleci/config.yml > deps.txt
```

Comment:
Could you tell us something more about this change?

Reply:
Before, the cache key that we generate would not be different if the `config.yml` changes, which can create some wasted cache hits when working on a PR like this that iterates on the caching and the config itself.